src/app/spaces/space.service.spec.ts

```diff
@@ -44,7 +44,13 @@ describe('Service: SpaceService', () => {
         {
           provide: WIT_API_URL,
           useValue: "http://example.com"
-        }
+        },
+        {
+          provide: AUTH_API_URL,
+          useValue: 'http://example.com/auth'
+        },
+        UserService,
+        Broadcaster
```

Comment:
Why did you import Broadcaster, I do see it being used. Additionally I'm not sure that you are using UserService either but at least it is used in another file.